assets/js/pages/UsersAndGroupsPage/PermEntry.js

```diff
@@ -10,19 +10,19 @@ const PermEntry = ({label, value, editable, allRoles, onDelete, onChange}) => {
       <span className="name">{label}</span>
       {
         editable ?
-        <PermSelect value={value} roles={allRoles} onChange={onChange} /> :
-        <span className="perm">{display}</span>
+          <PermSelect value={value} roles={allRoles} onChange={onChange} /> :
```

Comment:
Is this indentation part of ESLint rule as well? 
I actually like them head to head 😉